earth_enterprise/src/third_party/apache2/apache-gee-httpd-ssl.conf.in.patch

```diff
@@ -0,0 +1,108 @@
+--- httpd-2.2.32.orig/docs/conf/extra/httpd-ssl.conf.in	2015-05-27 11:59:59.000000000 -0700
```

Comment:
Yes. Also, if you click on the "Files Changed" tab and you see the file, then that means it will be in the repo after the PR is merged. If the second commit doesn't include a file, that doesn't mean the file is removed, just that the file isn't affected by that particular commit. If you want to remove the file, you'll need to do:
`
git rm filename
git commit
`

It looks like "/apache-gee-httpd.conf.in.patch" is also still included in this PR. That one should also be removed.